lib/main/autoUpdaterWrapper.ts

```diff
@@ -0,0 +1,104 @@
+import { app } from 'electron'
+import { autoUpdater } from 'electron-updater'
+import { mainWindow } from './app'
+
+export interface UpdateStatus {
+  updateAvailable: boolean
+  updateDownloaded: boolean
+}
+
+let updateStatus: UpdateStatus = {
+  updateAvailable: false,
+  updateDownloaded: false,
+}
+
+export function getUpdateStatus(): UpdateStatus {
+  return { ...updateStatus }
+}
+
+export function initializeAutoUpdater() {
+  // Initialize update status tracking
+  updateStatus = {
+    updateAvailable: false,
+    updateDownloaded: false,
+  }
+
+  // Allow auto-updater in development mode if VITE_DEV_AUTO_UPDATE is set
+  const enableDevUpdater = import.meta.env.VITE_DEV_AUTO_UPDATE === 'true'
```

Comment:
Only significant updates made (other than moving all auto updater related things to this file) were allowing for local dev updating.